CHANGELOG.md

```diff
@@ -1,4 +1,31 @@
-## 5.7.0 (Unreleased)
+## 5.7.0 (Nov 20, 2023)
```

Comment:
At top `## 5.8.0 (Unreleased)` is needed.

Reply:
Should that be in the main branch at this point? I have separately raised https://github.com/hashicorp/terraform-provider-google-beta/pull/6682 for the `release-5.8.0` branch.

Reply:
It is needed based on the doc https://github.com/hashicorp/terraform-provider-google/wiki/Release-Process#on-monday

Reply:
I think it is better to add it to avoid confusion in the future.